x-pack/plugins/logstash/server/routes/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { IRouter, ICustomClusterClient } from 'src/core/server';
+import { SecurityPluginSetup } from '../../../security/server';
+import { registerClusterLoadRoute } from './cluster';
+import {
+  registerPipelineDeleteRoute,
+  registerPipelineLoadRoute,
+  registerPipelineSaveRoute,
+} from './pipeline';
+import { registerPipelinesListRoute, registerPipelinesDeleteRoute } from './pipelines';
+import { registerUpgradeRoute } from './upgrade';
+
+export function registerRoutes(
+  router: IRouter,
+  esClient: ICustomClusterClient,
+  security?: SecurityPluginSetup
+) {
+  registerClusterLoadRoute(router, esClient);
+
+  registerPipelineDeleteRoute(router, esClient);
```

Comment:
Not sure if necessary as the routes seems to be heavily FTR covered, but would a unit test ensuring the routes get registered make any sense?

Reply:
Test should cover the logic at high risk, adding a route handler it's not one of them as it's already covered by the integration tests.